src/app.js

```diff
@@ -1,4 +1,6 @@
+const path = require('path');
 const express = require('express');
+var session = require('express-session')
```

Comment:
We can keep this if you add express-session as a dependency on the package.json file, else remove.